code/modules/mob/living/simple_animal/abnormality/teth/scorched_girl.dm

```diff
@@ -40,6 +40,29 @@
 	var/boom_damage = 250
 	patrol_cooldown_time = 10 SECONDS //Scorched be zooming
 
+	attack_action_types = list(
+		/datum/action/innate/change_icon_scorch
```

Comment:
preferably if you are doing multi-line lists, always have an object end with a comma. It makes future additions look better on github
```suggestion
		/datum/action/innate/change_icon_scorch,
```